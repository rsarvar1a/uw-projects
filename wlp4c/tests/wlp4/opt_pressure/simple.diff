2,4c2
< 65
< 252
< 20
---
> 1828
