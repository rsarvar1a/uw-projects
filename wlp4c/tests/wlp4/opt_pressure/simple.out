Enter length of array: Enter array element 0: Enter array element 1: Enter array element 2: Enter array element 3: Enter array element 4: Enter array element 5: Enter array element 6: Enter array element 7: Enter array element 8: Enter array element 9: Enter array element 10: Enter array element 11: Enter array element 12: Enter array element 13: Enter array element 14: Enter array element 15: Enter array element 16: Enter array element 17: Enter array element 18: Enter array element 19: Enter array element 20: Enter array element 21: Enter array element 22: Enter array element 23: Running MIPS program.
65
252
20
MIPS program completed normally.
$01 = 0x00000d08   $02 = 0x00000018   $03 = 0x00000000   $04 = 0x00000004   
$05 = 0x00000060   $06 = 0x00000000   $07 = 0x00000004   $08 = 0x00000008   
$09 = 0x00000000   $10 = 0x00000005   $11 = 0x00000001   $12 = 0x00000000   
$13 = 0x00000012   $14 = 0x00000000   $15 = 0x00000011   $16 = 0x00000000   
$17 = 0x00000010   $18 = 0x00000000   $19 = 0x0000000f   $20 = 0x00000000   
$21 = 0x0000000e   $22 = 0x00000000   $23 = 0x0000000d   $24 = 0x00000000   
$25 = 0x0000000c   $26 = 0x00000000   $27 = 0x00000000   $28 = 0x000005e4   
$29 = 0x01000000   $30 = 0x01000000   $31 = 0x8123456c   
